src/openms/source/ANALYSIS/TOPDOWN/PeakGroup.cpp

```diff
@@ -809,15 +861,14 @@ namespace OpenMS
     qvalue_[flag] = q;
   }
 
-  void PeakGroup::clearVectors()
+  void PeakGroup::clearPeaks()
```

Comment:
rename this to `deletePeaks`?
Clear suggests a 'clear()`, i.e. retain capacity IMHO.